authors/filip-srdoc/15-porezi/hr_HR/final.md

```diff
@@ -0,0 +1,107 @@
+
+U ostalim zemljama poput Australije, Singapura, Ujedinjenog Kraljevstva, Slovenije, Austriji i sličnih, način oporezivanja kriptovaluta ovisi o tome u kakvim transakcijama se kriptovaluta koristi, jesu li one privatne ili poslovne, koja je vrijednost valute u pitanju, na koji se način stekla valuta, je li to bila kupnjom, rudarenjem ili primanjem plaće i slično.
+
+Što se tiče Hrvatske, oporezivanje kriptovaluta također ovisi o gore navedenim radnjama pa se u skladu s njima koristi jedan od šest načina oporezivanja kapitalnog dohotka odnosno dobiti.
+
+Bilo je govora kako se zarada od rudarenja bitcoina uspoređuje s dobiti u igrama na sreću jer se za zatvaranje bloka transakcija u [blockchainu][bc] koji smo ovdje objasnili dobiva nagrada u obliku bitcoina. No, tu tvrdnju možemo odbaciti s obzirom da su [mining poolovi][mp] odnosno udruženi rudari sigurni da će izrudariti bitcoin pa to i nije neka definicija "igre na sreću."
+
+Kad bi osoba bila zaposlenik tvrtke koja rudari bitcoin bi li ona mogla primiti plaću u bitcoinu?
+
+Isti onaj ranije spomenuti porezni obveznik upitao je Poreznu upravu Republike Hrvatske o isplati plaća u kriptovalutama na što je PU odgovorila kako je kriptovalute moguće jedino isplatiti fizičkim osobama za radnje koje nisu obvezno isplative putem žiro računa fizičke osobe već ih je moguće isplatiti i u gotovini. To su mirovine, sezonski poslovi sa zaradom do 15 000 kn i sportske nagrade za nerezidente, ali ne i plaće. [Plaćom][placa] se smatraju svi primici koje poslodavac u novcu ili u naravi isplaćuje ili daje radniku po osnovi radnog odnosa. Ako poslodavac svom zaposleniku može unajmljivati stan čija vrijednost najma ulazi zaposleniku u bruto plaću te posljedično u osnovicu poreza na dohodak, zašto se onda tako ne bi tretirala i isplata plaća u kriptovalutama koje se preračunaju na određeni dan u nacionalnu valutu kao što je to slučaj u SAD-u i Australiji?
+
+### Rudarenje
+
+Kada bi fizička osoba primjerice rudarila bitcoin kod kuće za vlastite potrebe, to bi se smatralo drugim dohotkom koji inače ulazi u sustav sintetičkog oporezivanja dohotka zajedno s dohotkom od nesamostalnog rada i samostalne djelatnosti. Ako ukupni ostvareni prihod ne prelazi 210 000 kn porezna stopa je 24%, a ukoliko prelazi porezna stopa se penje na 36%.
```

Comment:
Ajmo malo prepast čitatelje ciframa. "To znači da na svaki izrudaredni bitcoin vi gubite 36% - drugim riječima, ako se bitcoin opet popne na 100,000kn, država će od vas uzimati 36,000 HRK za svaki bitcoin."